### ISSUE TITLE
🐛 BUG: Format on save issue

### ISSUE THREAD

**Issue by pgm-stevaelb** [withastro/language-tools#658]
### Describe the Bug

I am getting a strange, caching looking bug, when I try to format on save. This is the setting I have in my VSCode which before made it so that the code gets formatted on save:

```json
"[astro]": {
  "editor.defaultFormatter": "astro-build.astro-vscode"
},
```

For some reason, when I activate this code, and save when I just made the component, it formats back to that first save, every edit I do. When I disable that rule, it works, but my code isn't formatted on save and I have to manually format my code.

I have a link on the official Discord with a clip of what is happening: [https://discord.com/channels/830184174198718474/1155800328290177074/1156320306017882112](Discord link)

### Steps to Reproduce

https://discord.com/channels/830184174198718474/1155800328290177074/1156320306017882112

**Comment by Princesseuh** [comment]
Weird, never seen this before. Without a way to reproduce, it's hard to tell if the problem is in Prettier (the formatter we use internally), the way we call Prettier or something else.

**Comment by pgm-stevaelb** [comment]
> Weird, never seen this before. Without a way to reproduce, it's hard to tell if the problem is in Prettier (the formatter we use internally), the way we call Prettier or something else.

It's probably something like you say yes, do I need to share my config or something?

**Comment by Princesseuh** [comment]
Closing because I haven't had a single report of this in months, so I assume it got fixed in Volar 2.0. If this still happens, please open another issue (or mention it here)!
